python/lsst/ts/salobj/topics/read_topic.py

```diff
@@ -466,6 +470,7 @@ async def aget(self, timeout=None):
 
     def flush(self):
         """Flush the queue of unread data.
+        This flush will not affect the output of `aget`.
```

Comment:
Does it affect the output of `get` ? Presumably not?

Reply:
Correct. It affects neither of them. In https://github.com/lsst-ts/ts_salobj/pull/145 I changed the brief description to:

"Flush the queue used by `get_oldest` and `next`."

and left it at that. If you want more text, such as an explicit statement that data returned by `get` and `aget` is not affected, I can add that (but perhaps look at the new doc strings for those methods, first).